flake8_typing_imports.py

```diff
@@ -556,7 +556,7 @@ def parse_options(cls, options: Any) -> None:
         v = cls._min_python_version
         for part in cfg['options']['python_requires'].split(','):
             part = part.strip()
-            if part.startswith('>='):
+            if part.startswith('>=') or part.startswith('~='):
```

Comment:
`~=` is not the same as `>=` so this isn't correct -- `~=` can't correctly be represented in this scheme which depends on *minimum* version only, there is intentionally no upper bound

for example `~=3.6.4` would not match 3.7.0